packages/experimental/Shadow/SPEC.md

```diff
@@ -1,5 +1,45 @@
 # Shadow
 
 ## Background
+A `Shadow` is a component that can be used to add Fluent shadows to other components.
+
+Shadows defined in the Fluent design system are actually comprised of two shadows. The **key shadow** is used to create a feeling of distance, and the **ambient shadow** is used to define the edges of the shape. 
+
+## Shadow Depth Ramp
+
+These are the current `Shadow` variants:
+
+| | Light mode | Dark mode |
+| - | - | - |
+| Regular shadows | <img width="801" alt="Screen Shot 2022-08-12 at 10 46 45 AM" src="https://user-images.githubusercontent.com/78454019/184426272-fe23d23f-3c60-4811-82d8-16ae8dc501d9.png"> | <img width="788" alt="Screen Shot 2022-08-12 at 12 08 56 PM" src="https://user-images.githubusercontent.com/78454019/184427236-0b93d797-fdde-4367-9caa-751bcf335eea.png"> |
+| Brand shadows | <img width="802" alt="Screen Shot 2022-08-12 at 12 09 15 PM" src="https://user-images.githubusercontent.com/78454019/184427300-06dacedc-5f39-4dc1-b07b-536186115d2e.png"> | <img width="789" alt="Screen Shot 2022-08-12 at 12 09 03 PM" src="https://user-images.githubusercontent.com/78454019/184427338-c322d223-4719-4593-a550-8360c45aa2e5.png"> |
+
+## Sample Code
+
+Examples adding some Shadows to some text:
+
+```tsx
+<Shadow shadowToken={theme.shadows.shadow8}>
+  <Text>Text box with shadow8</Text>
+</Shadow>
+```
+```tsx
+<Shadow shadowToken={theme.shadows.shadow8brand}>
+  <Text>Text box with shadow8brand</Text>
+</Shadow>
+```
+For more examples of using Shadow, please see the [ShadowTest test page](https://github.com/microsoft/fluentui-react-native/tree/main/apps/fluent-tester/src/TestComponents/Shadow) in the [Fluent Tester app](https://github.com/microsoft/fluentui-react-native/blob/main/apps/fluent-tester/README.md).
+
+For an example of adding a Shadow as a slot to a Fluent component, please see the [FAB component](https://github.com/microsoft/fluentui-react-native/tree/main/packages/components/Button/src/FAB) (only on the iOS platform)
```

Comment:
Nit: iOS and Android. Also Notification is another example we can use but I think Fab night be a simpler example now that I think about it.